src/routes/docs/tutorials/nextjs/step-5/+page.markdoc

```diff
@@ -0,0 +1,278 @@
+---
+layout: tutorial
+title: Add database
```

Comment:
Maybe we can call this adding expenses? Since the overall is that we create a database so that we can create our expenses